bundles/jcr/repoinit/src/main/java/org/apache/sling/jcr/repoinit/impl/AclUtil.java

```diff
@@ -122,21 +197,60 @@ private static String toString(JackrabbitAccessControlEntry entry) throws Reposi
         private final Principal principal;
         private final Privilege[] privileges;
         private final boolean isAllow;
-        
+        private final LocalRestrictions oakRestrictions;
+
         LocalAccessControlEntry(Principal principal, Privilege[] privileges, boolean isAllow) {
+            this(principal, privileges, isAllow, null);
+        }
+
+        LocalAccessControlEntry(Principal principal, Privilege[] privileges,
+                                boolean isAllow, LocalRestrictions oakRestrictions) {
```

Comment:
Local means local to this class (inspired byLocalAccessControEntry :)  ) , just to separate from "Restriction"